index.html

```diff
@@ -321,25 +324,79 @@ <h1 class="contact-head-me">Contact me</h1>
               <p class="contact-text-body">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
               </p>
             </div>
-            <form action="https://formspree.io/f/xnqrezql" method="post" >
+            <form action="https://formspree.io/f/xnqrezql" method="post" class="contact-form" id="form" onload='document.form1.text1.focus()'>
               <ul class="form">
                 <li id="name-text-area ">
-                  <div id="name-basic-container"><input type="text" name="name" placeholder="Enter Full Name" maxlength="30" class= "name-text-input" required />
+                  <div id="name-basic-container"><input type="text" name="name" id="form_input1" placeholder="Enter Full Name" maxlength="30" class= "name-text-input" required />
                   </div>
                 </li>
                 <li id="mail-text-area">
-                  <div id="mail-basic-container"><input type="email" name="e-mail" placeholder="Enter e-mail" class="mail-text-input " required /></div>
+                  <div id="mail-basic-container"><input type="email" name="e-mail" id="form_input2" placeholder="Enter e-mail" class="mail-text-input " required /></div>
                 </li>
                 <li id="text-area">
                   <textarea name="text-area" placeholder="Write your message here" cols="30" rows="10" maxlength="500" class="txt-area txt-cont-out" required></textarea>
                 </li>
               <li class="action">
-                <button type="submit" class="btn-sz-sub sub_text">Get In Touch</button>
+                <button type="submit" id="validate" class="btn-sz-sub sub_text">Get In Touch</button>
```

Comment:
- [ ] According to the requirements, there should be an error message that shows up close to the button when a user tries to submit the form if the email is not all lowercase. Please kindly implement this.